### PR TITLE
key format fix

### PR DESCRIPTION
**Pull request by rjt-gupta**
I think the key for `smembers` command should be of this format.

Refs - https://github.com/aio-libs/aioredis/blob/e8c33e39558d4cc91cf70dde490d8b330c97dc2e/tests/set_commands_test.py#L190

### CODE REVIEW

**Comment by rjt-gupta** [comment]
#316 needs this change so, please have a look at this first.

File: tanner/api/api.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def __init__(self, redis_client):
     async def return_snares(self):
         query_res = []
         try:
-            query_res = await self.redis_client.smembers('snare_ids', encoding='utf-8')
+            query_res = await self.redis_client.smembers('key:snare_ids', encoding='utf-8')
```
Comment (rnehra01) [inline tanner/api/api.py]: The previous one is working fine for me. Did it give some error or return the wrong results?

Comment (rjt-gupta) [reply tanner/api/api.py]: I tried it without `key:` and error I got - `aioredis.errors.ReplyError: WRONGTYPE Operation against a key holding the wrong kind of value`. Also, while testing using `redis_client.sadd()` gave me same error without using `key:`.

Comment (rjt-gupta) [reply tanner/api/api.py]: Wait same thing I tried [here](https://github.com/mushorg/tanner/blob/d7b6c4b3bb0497941d12b4747fae196f5accd604/tanner/api/api.py#L45) but it works both ways. Let me check this again.

Comment (rjt-gupta) [reply tanner/api/api.py]: Nope `smembers` not working.

**Comment by rjt-gupta** [comment]
So, I think the problem is key `snare_ids` is mapped to values of type `string` (checked through redis-cli). Everything else is of type `set`. And for type `string` we use commands such as `get`, to use `smembers`/`sadd` the key should be of type `set`.

**Comment by rjt-gupta** [comment]
And Its not an issue anymore :)

**Pull request closed without merging**